src/main.cpp

```diff
@@ -33,9 +33,8 @@
 
 #include "query/parsererrors.h"
 #include <pdaaal/PDAFactory.h>
-#include "pdaaal/engine/Moped.h"
-#include <pdaaal/Solver.h>
-
+#include <pdaaal/Solver_Adapter.h>
```

Comment:
Is this updated according to PDAAAL?

Reply:
No, we need to use the new version of PDAAAL. 
I'll wait for https://github.com/DEIS-Tools/PDAAAL/pull/3, then make PDAAAL release v0.2, and then update this pull request to use that release.

Reply:
Commit (bb94285) solves this.